web/oidc/oidc.go

```diff
@@ -32,6 +32,13 @@ import (
 	"github.com/labstack/echo/v4"
 )
 
+var (
+	ErrInvalidToken         = errors.New("invalid token")
+	ErrInvalidConfiguration = errors.New("invalid configuration")
+	ErrAuthenticationFailed = errors.New("the authentication has failed")
```

Comment:
For this error, the string is used as a key for localization. So, we need to keep the same string, or update the locale files.

Reply:
ok, I fix this

Reply:
I have updated the locals for `ErrAuthenticationFailed`. The others have not changed. Everything should be good